docs/library/platform.rst

```diff
@@ -0,0 +1,28 @@
+:mod:`platform` -- access to underlying platform’s identifying data
+===================================================================
+
+.. module:: platform
+   :synopsis: access to underlying platform’s identifying data
+
+|see_cpython_module| :mod:`python:platform`.
+
+This module tries to retrieve as much platform-identifying data as possible. It
+makes this information available via function APIs.
+
+Functions
+---------
+
+.. function:: platform()
+
+   Returns a single string identifying the underlying platform. For example,
+   this could be ``"Linux"``, ``"Unix"`` or ``"Windows"``.
```

Comment:
According to the source code, this is going to be something really long with a lot of dashes.

https://github.com/micropython/micropython/blob/a18d62e06727e0c424da1f43f80a5b0cb76bcc04/extmod/modplatform.c#L39C51-L41C59